Automattic/wp-calypso
# Extract `create-config` into `@automattic/create-calypso-config` and convert to TypeScript and jest

#### Changes proposed in this Pull Request

* Extracts `create-config` into a new `create-calypso-config` package.

#### Testing instructions



* Ensure calypso configuration stuff continues to run smoothly and as expected. 
* Ensure unit and e2e tests pass
